lib/logstash/version.rb

```diff
@@ -1,6 +1,6 @@
 # encoding: utf-8
 # The version of logstash.
-LOGSTASH_VERSION = "2.0.0.dev"
+LOGSTASH_VERSION = "1.5.0.rc3"
```

Comment:
I guess the right point to change this variable is within the 1.5.x branch, not in here as master for now is target to be another thing (even is for now is the same as 1.5, for last rc2 we did the change just in the 1.5 branch)

We might need to review our branching strategy as for now 2.0 and 1.5 branches are basically similar or very close by.